src/bls12_381/fq.rs

```diff
@@ -1,9 +1,16 @@
-use ::{Field, PrimeField, SqrtField, PrimeFieldRepr, PrimeFieldDecodingError};
+use {Field, PrimeField, PrimeFieldDecodingError, PrimeFieldRepr, SqrtField};
```

Comment:
Not a fan of requiring imports to be sorted; I see the motivation, but it prevents you from grouping logically related imports.

Reply:
Logically related imports can be brought in with different `use` statements.